packages/server/docs/openapi.yaml

```diff
@@ -0,0 +1,1933 @@
+      summary: 'Get all categories.'
+      tags:
+        - category
+      operationId: categoriesGet
+      responses:
+        '200':
+          description: Categories.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  categories:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/CategoryDocument'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /eventlogs:
+    post:
+      description: 'API endpoint to get a list of Event Logs. Ordered from newest to oldest.'
+      summary: 'API endpoint to get a list of Event Logs.'
+      security:
+        - requireAdmin: []
+      tags:
+        - eventlogs
+      operationId: eventlogsPost
+      requestBody:
+        description: Which eventlogs to return.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PaginateEventLogsPost'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/PaginateEventLogsPost'
+      responses:
+        '200':
+          description: Event logs
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  estimatedTotal:
+                    $ref: '#/components/schemas/MetaCount'
+                  eventLogs:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        _id:
+                          $ref: '#/components/schemas/Meta_id'
+                        actor:
+                          $ref: '#/components/schemas/MetaUser'
+                        detail:
+                          type: object
+                          properties:
+                            kind:
+                              $ref: '#/components/schemas/MetaEventKind'
+                            propertyNew:
+                              type: string
+                              description: New value for update type events.
+                            propertyOld:
+                              type: string
+                              description: Old value for update type events.
+                        kind:
+                          $ref: '#/components/schemas/MetaEventKind'
+                        createdAt:
+                          $ref: '#/components/schemas/MetaDateTime'
+                        updatedAt:
+                          $ref: '#/components/schemas/MetaDateTime'
+                        __v:
+                          type: integer
+                          default: 0
+        '401':
+          $ref: '#/components/responses/ErrorAuthentication'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /hotlines:
```

Comment:
We don't currently have hotlines in our DB, or at least we're not using this endpoint to reach them. Thoughts on removing this and reapplying when we are, or just leaving as-is and updating once we get to using it?

Reply:
I vote we remove it until we have a need for a hotlines endpoint 👍🏼

Reply:
remove just the doc and leave the endpoint?

If that's the case

Maybe add a note in the summary / description / tag that it's not currently used

Alternatively could remove the whole endpoint from the api?

Reply:
for now I commented out the `/hotlines` endpoint

Reply:
> for now I commented out the /hotlines endpoint

Yep, I like that plan!